Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -766,6 +812,11 @@ def parse_table_row_to_dict(alert: list[dict[str, Any]]) -> dict[str, Any]:
                 obj[key] = int(obj[key])
             except Exception:
                 demisto.info(f"Could not parse risk level {obj[key]} to int")
+        if key in ["group", "job_title", "departments", "labels"]:
+            try:
+                obj[key] = json.loads(obj[key])
```

Comment:
What happens when the key is labels? If you don't replace the "", an error will occur when calling to json.loads(obj[key]).